### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jiaopi404**

### CODE REVIEW

**Comment by jiaopi404** [comment]
Hi Jing. Can you see me @xujing0521

**Comment by jiaopi404** [comment]
Re-ping @xujing0521

File: README.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ yarn release    // bump version for release
 yarn pub        // publish
 ```
 
+## From Joel
+
+Hi Jing
```
Comment (xujing0521) [inline README.md:20]: 你很皮哦, say you love me

**Review by xujing0521 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jiaopi404) [reply README.md:20]: you love me

Comment (jiaopi404) [reply README.md:20]: 同事不会看到吧

**Pull request closed without merging**